src/sdk/main/include/TokenUpdateNftsTransaction.h

```diff
@@ -0,0 +1,171 @@
+/*-
+ *
+ * Hedera C++ SDK
+ *
+ * Copyright (C) 2020 - 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+#ifndef HEDERA_SDK_CPP_TOKEN_UPDATE_NFTS_TRANSACTION_H_
+#define HEDERA_SDK_CPP_TOKEN_UPDATE_NFTS_TRANSACTION_H_
+
+#include "AccountId.h"
+#include "TokenId.h"
+#include "Transaction.h"
+
+#include <chrono>
+#include <memory>
+
+namespace proto
+{
+class TokenUpdateNftsTransactionBody;
+class TransactionBody;
+}
+
+namespace Hedera
+{
+
+class TokenUpdateNftsTransaction : public Transaction<TokenUpdateNftsTransaction>
```

Comment:
Should we have a comment describing the class and what it does?

Reply:
Will submit this with next commits.